environment/src/main/java/com/sequenceiq/environment/environment/domain/EnvironmentView.java

```diff
@@ -68,6 +74,122 @@
     @Convert(converter = EnvironmentStatusConverter.class)
     private EnvironmentStatus status;
 
+    @Convert(converter = EnvironmentDeletionTypeConverter.class)
```

Comment:
I'm sure N+1 issue is solved but now everywhere we use this view the amount of data read from the database will increase because this is not remotely a lightweight view anymore.